docs/topics/feed-exports.rst

```diff
@@ -267,10 +267,140 @@ once all the file contents have been written (i.e. at the end of the crawl) is
 that file uploaded to the feed URI.
 
 If you want item delivery to start earlier when using one of these storage
-backends, use :setting:`FEED_EXPORT_BATCH_ITEM_COUNT` to split the output items
-in multiple files, with the specified maximum item count per file. That way, as
-soon as a file reaches the maximum item count, that file is delivered to the
-feed URI, allowing item delivery to start way before the end of the crawl.
+backends, use :ref:`batches<batches>` to split the output items in multiple
+files, based on a specified limit. That way, as soon as a file reaches the specified
+limit, that file is delivered to the feed URI, allowing item delivery to start
+way before the end of the crawl.
+
+.. _batches:
+
+Batches
+=======
+
+.. versionadded:: VERSION
+
+Batches enable multiple sequential files to be generated for a feed instead of getting
+a single file. This is useful in getting item deliveries earlier before the crawl has
+been ended.
+
+To enable batching, you can use :ref:`criterias<batch_criterias>` such as ``batch_item_count``
+, ``batch_file_size``, ``batch_duration`` or you can create :ref:`your own batch
+handlers<custom_batch_handler>` to handle batch deliveries.
+
+When generating multiple output files, you must use at least one of the following
+placeholders in the feed URI to indicate how the different output file names are
+generated:
```

Comment:
You can probably replace this whole part about batch feed URIs with a paragraph that simply states that, if you enable batching, you must update your feed URI accordingly, and then link to a new section that contains all the information here.

This is to avoid the most part of the introduction about batches being something as specific as the handling of the feed URI. It also allows us to link to this information from other parts of the documentation and from the outside.